src/processing/dom/index.js

```diff
@@ -209,6 +216,13 @@ export default class DomProcessor {
         this.adapter.setAttr(el, 'autocomplete', 'off');
     }
 
+    // NOTE: We just remove the 'integrity' attribute, because its value is not relevant after the script content
+    // is changed (http://www.w3.org/TR/SRI/). If this causes problems in the future, we will need to generate
+    // the correct SHA for the changed script.
```

Comment:
already checked, but there's nothing that can't be improved
```
    // NOTE: We simply remove the 'integrity' attribute because its value will not be relevant after the script content
    // changes (http://www.w3.org/TR/SRI/). If this causes problems in the future, we will need to generate
    // the correct SHA for the changed script.
```